user-manuals/overworld/README.md

```diff
@@ -3,28 +3,37 @@
 The overworld is the core of the Gamify-IT platform.
 It is the main interface for students to interact with the platform.
 
-The overworld consists of multiple regions and dungeons.
-Throughout the world, students can discover course contents in form of [minigames](../README.md) and NPC dialogues.
+## Overview
+
+The overworld consists of multiple `worlds` and `dungeons`.
+Throughout these areas students can discover course contents like playable [minigames](../README.md) and dialogues with `NPCs`. The player (the character in the middle of the picture) can walk around and interact with various objects such as the NPC on the left or `books` providing useful information. The `HUD` helps navigating with the minimap and keeping track of the progress with the progress bar.  
 ![overworld](../../images/overworld.webp)
 
-- The player is in the center of the screenshot
-- They can walk around and interact with various objects such as the NPC on the left.
-- There is a HUD which displays a minimap, and other useful information
+## Gameplay
+
+### NPCs
 
+The player can talk to other characters in the world to obtain information. The lecturer creating the course can specify what each `NPC` has to say and therefor give tips and summaries of important topics to the player.
 ![npc dialogue](../../images/npc-dialogue.webp)
 
-- the lecturer can configure what the NPCs say
-- they can use that to give tips about a topic for example
+### Books
+
+For large amounts of information, there are also `books` to find. Those are very similar to NPC, they can also be configured by the lecturer and are meant to provide assistance. But unlike NPCs, they don't show multiple sentences but one big page. With that it is possible for the course creator to choose between different ways to display information and help the player.  
```

Comment:
```suggestion
For large amounts of information, you can use books.  
They are similar to NPCs:
They can also be configured by the lecturer and are meant to provide assistance.  
Unlike NPCs, however, they show all content combined on one big page instead of splitting it up into sentences.  
Hence, there are different ways to display information and help the player.  
```